k-distribution/tutorial/2_languages/1_simple/1_untyped/simple-untyped.md

```diff
@@ -1,114 +1,129 @@
-// Copyright (c) 2014-2019 K Team. All Rights Reserved.
+---
+copyright: Copyright (c) 2014-2020 K Team. All Rights Reserved.
+---
 
-/*!
-\title{SIMPLE --- Untyped}
-\author{Grigore Ro\c{s}u and Traian Florin \c{S}erb\u{a}nu\c{t}\u{a}
-        (\texttt{\{grosu,tserban2\}@illinois.edu})}
-\organization{University of Illinois at Urbana-Champaign}
-*/
+SIMPLE — Untyped
+================
 
-/*@ \section{Abstract}
-This is the \K semantic definition of the untyped SIMPLE language.
+Author: Grigore Roșu (grosu@illinois.edu)  
+Organization: University of Illinois at Urbana-Champaign
+
+Author: Traian Florin Șerbănuță (traian.serbanuta@unibuc.ro)  
+Organization: University of Bucharest
+
+
+### Abstract
+This is the **K** semantic definition of the untyped SIMPLE language.
 SIMPLE is intended to be a pedagogical and research language that captures
 the essence of the imperative programming paradigm, extended with several
 features often encountered in imperative programming languages.
 A program consists of a set of global variable declarations and
 function definitions.  Like in C, function definitions cannot be
-nested and each program must have one function called \texttt{main},
+nested and each program must have one function called `main`,
 which is invoked when the program is executed.  To make it more
-interesting and to highlight some of \K's strengths, SIMPLE includes
+interesting and to highlight some of **K**'s strengths, SIMPLE includes
 the following features in addition to the conventional imperative
 expression and statement constructs:
-\begin{itemize}
-\item Multidimensional arrays and array references.  An array evaluates
+
+* Multidimensional arrays and array references.  An array evaluates
 to an array reference, which is a special value holding a location (where
 the elements of the array start) together with the size of the array;
 the elements of the array can be array references themselves (particularly
 when the array is multi-dimensional).  Array references are ordinary values,
 so they can be assigned to variables and passed/received by functions.
```

Comment:
I prefer bullet lists to be indented more like this (for readability/scanability):

```suggestion
* Multidimensional arrays and array references.  An array evaluates
  to an array reference, which is a special value holding a location (where
  the elements of the array start) together with the size of the array;
  the elements of the array can be array references themselves (particularly
  when the array is multi-dimensional).  Array references are ordinary values,
  so they can be assigned to variables and passed/received by functions.
```

Reply:
I would also prefer four spaces instead of 2, but up to you.